wordpress-mobile/WordPress-iOS
# Issue/8306 empty my sites

Fixes #8306 

Needs Review: @nheagy, @folletto 

**To test:** 
- Log into an account with no sites, or delete all sites from an account.
- Verify new view is shown.

**Designed:**

<img width="250" alt="my_sites_designed" src="https://user-images.githubusercontent.com/1816888/34063817-da81a734-e1b1-11e7-9d1e-634c5f7ca50e.png">

**Implemented:**

![my_sites_implemented](https://user-images.githubusercontent.com/1816888/34063878-4081ae4e-e1b2-11e7-899b-5f142547f30a.png)

![implemented_landscape](https://user-images.githubusercontent.com/1816888/34063914-9044d33e-e1b2-11e7-9343-465bc4ac85bf.png)
